.golangci.yml

```diff
@@ -0,0 +1,26 @@
+output:
+  sort-results: true
+
+run:
+  build-tags:
+  - withebpf
+  - tools
+  - docs
+  skip-files:
+  # Skip autogenerated files.
+  - ^.*.pb.go$
+
+issues:
+  exclude-use-default: false
+  max-same-issues: 0
+  max-issues-per-linter: 0
+  exclude-rules:
+  # Ignore autogenerated code.
+  - linters:
+    - gofumpt
+    path: pkg/client/clientset
```

Comment:
IIUC, this will prevent `gofumpt` to be run on `pkg/client/clientset`, isn't it?

If so, I would prefer to use this option instead of using `run->skip-{dirs|files}` which analyzes the files but doesn't report the issues, [check here](https://github.com/golangci/golangci-lint/blob/29eedbf6e56508cf1ca52ad0eeed067b1c3a3066/.golangci.example.yml#L36-L44
). What do you think?

Reply:
Actually, gofumpt does not complain about files in `pkg/client/clientset`, perhaps it detects that those files are autogenerated somehow, so it can be removed.